docs/docs_source/wopi/files/RefreshLock.rst

```diff
@@ -14,17 +14,35 @@ RefreshLock
     The refreshed lock must expire automatically after 30 minutes unless it is modified by a subsequent WOPI
     operation, such as :ref:`Unlock` or :ref:`RefreshLock`.
 
+    ..  include:: /fragments/priorlock.rst
+
+    ..  include:: /fragments/lock409.rst
+
+    ..  include:: /fragments/no_lock_id.rst
+
+    ..  note::
+        Since there is no GetLock operation defined in WOPI, Office Online will sometimes make a RefreshLock call to
```

Comment:
Is that always the case across all three apps? I suppose we could technically use any lock-related op for that purpose since they're all supposed to return the current lock, right?